lib/livebook/session.ex

```diff
@@ -508,6 +519,21 @@ defmodule Livebook.Session do
     end
   end
 
+  @impl true
+  def handle_call(:infer_file_name, _from, state) do
```

Comment:
We don't need to call the server, instead we can just pass the session struct to the function, like `Session.file_name_for_download(session)` :)